README.md

```diff
@@ -39,7 +39,7 @@ git clone <your fork>
 cp env-example .env
 bundle
 yarn
-bundle exec rails s
+bundle exec rspec 
```

Comment:
Above are the instructions to start server locally, may be leave `bundle exec rails s` here and add `bundle exec rspec` as well with some comments saying "run rspec tests".

Reply:
@vinithanatarajan Yes my bad should be `bundle exec rspec` only in the testing section : }